livemaps-api/APIFunctions/AtlasUnits.cs

```diff
@@ -1,34 +1,39 @@
 using System;
-using System.IO;
+using System.Net.Http;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
-using System.Net.Http;
-using ssir.api.Models.Atlas;
+using Ssir.Api.Models.Atlas;
 
-namespace ssir.api.APIFunctions
+namespace Ssir.Api.APIFunctions
 {
     public static class AtlasUnits
     {
         [FunctionName("AtlasUnits")]
         public static async Task<IActionResult> Run(
+            //This HTTP trigger sends a client HTTP request through the atlasSubscriptionKey and atlasDataSetId parameters to get featureIds.
             [HttpTrigger(AuthorizationLevel.Function, "get", "post", Route = null)] HttpRequest req,
             ILogger log)
         {
+            //Get the atlasSubscriptionKey value from environment variable.
             var atlasSubscriptionKey = Environment.GetEnvironmentVariable("atlasSubscriptionKey");
+
+            //Get the atlasDataSetId value from environment variable.
             var atlasDataSetId = Environment.GetEnvironmentVariable("atlasDataSetId");
 
             using (var client = new HttpClient())
             {
                 var baseUri = new System.Text.StringBuilder($"https://atlas.microsoft.com/wfs/datasets/{atlasDataSetId}/collections/unit/items?api-version=1.0&subscription-key={atlasSubscriptionKey}&limit=2");
```

Comment:
This should be using `UriBuilder` rather than treating the URI as a a string.

Reply:
@jsquire I have updated to UriBuilder, please review.